spec/decorators/user_decorator_spec.rb

```diff
@@ -72,6 +72,97 @@
     end
   end
 
+  describe '#pending_profile_requires_verification?' do
+    it 'returns false when no pending profile exists' do
+      profile = create(:profile)
+      user_decorator = UserDecorator.new(profile.user)
+
+      expect(user_decorator.pending_profile_requires_verification?).to eq false
+    end
+
+    it 'returns true when no active profile exists' do
```

Comment:
Also, what do you think about using `build_stubbed` instead of `create` for the profiles to speed up the tests?